orchagent/orch.cpp

```diff
@@ -133,6 +140,48 @@ void Consumer::drain()
         m_orch->doTask(*this);
 }
 
+// TODO: Table should be const
+void Orch::addExistingData(Table *table)
```

Comment:
can you add one line description what addExistingData do?

Reply:
Actually it is in the header file.

---
In reply to: [205862889](https://github.com/Azure/sonic-swss/pull/551#discussion_r205862889) [](ancestors = 205862889)